### PR TITLE
Still some more places with config stuff

### PR DESCRIPTION
**Pull request by annando**
See the PR I did before.

Accidentally some small other stuff was added - but this doesn't harm anything.

### CODE REVIEW

File: view/theme/quattro/style.php [hunk 1]
```diff
@@ -5,27 +5,30 @@
...
 	$uid = get_theme_uid();
 
-	$color=false;
-	$quattro_align=false;
-	$site_color = Config::get("quattro","color");
-	$site_quattro_align = Config::get("quattro", "align" );
-	
+	$color = false;
+	$quattro_align = false;
+	$site_color = Config::get("quattro", "color", "dark);
```
Comment (MrPetovan) [inline view/theme/quattro/style.php]: A double-quote is missing here.

File: view/theme/quattro/style.php [hunk 2]
```diff
@@ -5,27 +5,30 @@
...
-	if ($color===false) $color=$site_color;
-	if ($color===false) $color="dark";
-	if ($quattro_align===false) $quattro_align=$site_quattro_align;
-	
-		
+		$color = PConfig::get($uid, "quattro", "color");
+		$quattro_align = PConfig::get($uid, 'quattro', 'align');
+	}
+
+	if ($color === false) {
```
Comment (MrPetovan) [inline view/theme/quattro/style.php]: Isn't this supposed to be `null` if the `PConfig::get` above call can't find the quattro/color config value?

File: view/theme/quattro/style.php [hunk 3]
```diff
@@ -5,27 +5,30 @@
...
+		$color = $site_color;
+	}
+
+	if ($quattro_align === false) {
```
Comment (MrPetovan) [inline view/theme/quattro/style.php]: Same false/null question here.

File: mod/settings.php [hunk 4]
```diff
@@ -978,24 +978,14 @@ function settings_content(App $a) {
...
-
-		$no_auto_update = PConfig::get(local_user(), 'system', 'no_auto_update');
-		$no_auto_update = (($no_auto_update===false)? '0': $no_auto_update); // default if not set: 0
-
-		$bandwidth_saver = PConfig::get(local_user(), 'system', 'bandwidth_saver');
-		$bandwidth_saver = (($bandwidth_saver === false) ? '0' : $bandwidth_saver); // default if not set: 0
+		$noinfo = PConfig::get(local_user(), 'system', 'ignore_info', 0);
+		$infinite_scroll = PConfig::get(local_user(), 'system', 'infinite_scroll', 0);
+		$no_auto_update = PConfig::get(local_user(), 'system', 'no_auto_update', 0);
+		$bandwidth_saver = PConfig::get(local_user(), 'system', 'bandwidth_saver', 0);
```
Comment (MrPetovan) [inline mod/settings.php]: Nice job condensing those calls!

File: mod/admin.php [hunk 5]
```diff
@@ -2273,10 +2273,7 @@ function admin_page_features(App $a) {
 			$arr[$fname][0] = $fdata[0];
 			foreach (array_slice($fdata,1) as $f) {
 
-				$set = Config::get('feature',$f[0]);
-				if ($set === false) {
-					$set = $f[3];
-				}
+				$set = Config::get('feature',$f[0], $f[3]);
```
Comment (MrPetovan) [inline mod/admin.php]: Standards: Can you please add a space after the first comma?

**Comment by annando** [comment]
@MrPetovan Done.

File: mod/admin.php (hunk 5 above)
Comment (MrPetovan) [reply mod/admin.php]: You added spaces on line 2237 but not on line 2267!

File: view/theme/quattro/style.php (hunk 3 above)
Comment (annando) [reply view/theme/quattro/style.php]: This is okay, since the default value is set to "false".

**Comment by annando** [comment]
And once again

**Comment by MrPetovan** [comment]
Didn't you sleep at all last night? 😯

**Comment by annando** [comment]
According to my watch I slept 2:44. But I'm not tired. Normally I sleep 5-6 hours.

**Comment by MrPetovan** [comment]
Wow, I slept 6 full hours and I feel tired!

**Pull request merged**